api/src/routers/meals.js

```diff
@@ -0,0 +1,75 @@
+import express from "express";
+import knex from "../database_client.js";
+
+const mealsRouter = express.Router();
+
+//return all meals
+mealsRouter.get("/", async (req, res, next) => {
+  try {
+    const meals = await knex("Meal").select("*");
+    res.json(meals);
+  } catch (error) {
+    next(error);
+  }
+});
+
+//insert meal
+mealsRouter.post("/", async (req, res, next) => {
+  try {
+    console.log(req.body);
+    const data = req.body;
+    await knex("Meal").insert(data);
+    res.status(200).json({ message: "created successfully" });
+  } catch (error) {
+    next(error);
+  }
+});
+
+//GET meals by id
+mealsRouter.get("/:id", async (req, res, next) => {
+  try {
+    const id = req.params.id;
+    const meal = await knex("Meal").select("*").where("id", id);
```

Comment:
We expect an object but `where` returns a list. Also, it's returning an empty list when the meal is not found, not 404.

Add `first` at the end to return one object.

```suggestion
    const meal = await knex("Meal").where("id", id).first();
```